.jenkinsci/linux-post-step.groovy

```diff
@@ -1,6 +1,23 @@
 def linuxPostStep() {
   timeout(time: 600, unit: "SECONDS") {
     try {
+      // stop write core dumps
+      sh "bash -c 'ulimit -c 0'"
+      if (currentBuild.currentResult != "SUCCESS") {
+        // handling coredumps (if tests crashed)
+        def currentPath = sh(script: "pwd", returnStdout: true).trim()
+        def dumpsFileName = sprintf('coredumps-%1$s.tar',
+          [GIT_COMMIT.substring(0,8)])
+
+        // sh(script: "find ${currentPath} -type f -name '*.coredump' | zip -j ${dumpsFileName} -@")
+        sh(script: "find ${currentPath} -type f -name '*.coredump' -exec tar -rvfj ${dumpsFileName} {} \\;")
```

Comment:
Please, flatten directory structure in an archive. So that it only contains files, not directory trees.

Reply:
Why should we flatten the directories? I will only complicate the code (which is now very readable) and cause possible name conflicts resulting in dumps being overwritten or skipped. I do not think it is worth a couple of times pressing \<Tab\> to auto-complete the path to core dump in a shell. Also the original directory tree will make it easier to understand which executable it corresponds to.

Reply:
If the way coredumps are stored in an archive is OK with other developers then we can keep it